README.md

```diff
@@ -311,7 +330,8 @@ Tips for Fixture Loading Tests
     **Attention: you need Doctrine >= 2.2 to use this feature.**
 
     ```yaml
-    # app/config/config_test.yml
+    # sf3: app/config/config_test.yml
+    # sf4: config/packages/test/framework.yaml
     liip_functional_test:
         cache_db:
             sqlite: liip_functional_test.services_database_backup.sqlite
```

Comment:
This parameter has been renamed to `cache_sqlite_db` in the 2.x version of this bundle, see the upgrade guide : https://github.com/liip/LiipFunctionalTestBundle/blob/d3f3452aa7c2a82d2316dc0042602a8ba08955ed/UPGRADE-2.0.md